src/modules/game.js

```diff
@@ -0,0 +1,37 @@
+import { setKey, getKey } from './keys.js';
+import { setScore, getScore } from './scores.js';
+import render from './display.js';
+
+const createNewGame = async () => {
+  const game = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json;charset=utf-8',
+    },
+    body: JSON.stringify({ name: 'MicroGameXtrem' }),
+  });
+  return game.json().then((res) => {
+    const key = toString(/\w{20}/.exec(res.result)[0]);
+    setKey(key);
+  });
+};
+
+const fetchScores = async () => {
+  const res = await fetch(`https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${getKey()}/scores/`);
+  return res.json().then(({ result }) => {
+    setScore(result);
+    render(getScore());
+  });
```

Comment:
:red_circle: : [REQUIRED] I think the then function called on the second promise (Line 21) should receive the result as an argument rather than an object. Your code should look like this :arrow_down: 

```
 return res.json().then(( result ) => {
    setScore(result); //  **Note: I think you don't need this line. Please remove it** 
    render(result); // Note: You can pass directly the result to your render method
  });
```

Reply:
> Correction

The complexity was eliminated and the result is passed directly